src/Microsoft.Health.Fhir.Liquid.Converter/Json/DictionaryJsonConverter.cs

```diff
@@ -5,10 +5,9 @@
 
 using System;
```

Comment:
This file seems a base class, and `C-CDA` inherits it, is it appropriate to put it here?

Reply:
`DictionaryJsonConverter` is the base class of `XmlDictionaryJsonConverter`. The former one is used in JSON-to-FHIR to parse generic JSON data to dictionary directly. Based on that, the latter one has some extra logics to parse the XML-converted JSON to work with the CCDA templates. I've moved `DictionaryJsonConverter` to namespace `Converter` instead of `Converter.Json` in the update.